### PR TITLE
refactor: minor updates

### PR DESCRIPTION
**Pull request by minh-rakuten**
# Description
- Change `UserInfoBridgeDispatcher` into interface.
- Remove application context in `RealMiniAppDisplayer` due to no usage.

# Checklist
- [x] I have read the [contributing guidelines](../.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](../.github/CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

File: testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt [hunk 1]
```diff
@@ -160,7 +160,7 @@ class MiniAppDisplayActivity : BaseActivity() {
         miniAppMessageBridge.setAdMobDisplayer(AdMobDisplayer(this@MiniAppDisplayActivity))
         miniAppMessageBridge.allowScreenOrientation(true)
 
-        val userInfoBridgeDispatcher = object : UserInfoBridgeDispatcher() {
```
Comment (khairul-alam-licon) [inline testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt:163]: We need to update the userguide as well according to this change.

**Review by khairul-alam-licon — APPROVED** [review]
Looks good 👍

File: testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt (hunk 1 above)
Comment (minh-rakuten) [reply testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt:163]: updated.

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,7 @@
 Replace `getUserName`, `getProfilePhoto` with new prototypes.
 
 - **Change:** Support Android 7 - API 24 as minimum version.
+- **Change:** Convert `UserInfoBridgeDispatcher` into interface.
```
Comment (corycaywood) [inline CHANGELOG.md:16]: Could you also add a note about how they should update their code? i.e. something like: 
```suggestion
- **Change:** Convert `UserInfoBridgeDispatcher` into interface. Usages in your code of `object : UserInfoBridgeDispatcher()` should be changed to `object : UserInfoBridgeDispatcher`.
```

**Review by corycaywood — APPROVED** [review]
Looks good 👍  Just left a suggestion about the change log.

File: CHANGELOG.md (hunk 2 above)
Comment (minh-rakuten) [reply CHANGELOG.md]: updated.

**Pull request merged**